base_layer/tari_stratum_ffi/src/lib.rs

```diff
@@ -280,41 +280,84 @@ mod tests {
-    const HASH_HEX: &str = "3a9ea717ca7b2598d900e2ef98c270ac98ce993bce8a9e058929967ba37fbc6b";
-    const NONCE: u64 = 15810795621223647638;
+    // Hex representation of block template ("blocktemplate_blob"). A new one can be retrieved by querying the
+    // getblocktemplate method of tari_stratum_transcoder when needed. This can be done with Postman by setting up a
+    // POST request with the following RAW body: {"id":"0","jsonrpc":"2.0","method":"getblocktemplate"} Note that if
+    // this hex string needs to be changed then a new DLL (Windows) and SO (Linux) library will need to be built and
+    // committed to the Miningcore codebase since the structure of the template or the underlying data has changed.
+    // TODO: Fix tari_stratum_transcoder and get a valid block template hex representation
+    const BLOCK_HEX: &str = "";
+
+    // Nonce for the block. Randomly selected, difficulty of at least 10 should be sufficient.
+    // TODO: Determine a nonce that will provide an adequate difficulty for these test
+    const NONCE: u64 = 995572868245622544;
+
+    // Hash of the block template header AFTER the nonce was injected into it.
+    // TODO: Fix tari_stratum_transcoder and get a valid header hex hash
+    const HASH_HEX: &str = "";
+
+    #[test]
+    #[ignore = "utility used for test setup"]
+    // The following function can be used to get the hash for these tests
+    // run with --nocapture flag to see output
+    fn get_hash() {
+        let block_hex = hex::decode(BLOCK_HEX).unwrap();
+        let mut block: Block = serde_json::from_str(&String::from_utf8_lossy(&block_hex).to_string()).unwrap();
+        block.header.nonce = NONCE;
+        let hash_hex = block.header.hash().to_hex();
+        println!("{}", hash_hex);
+    }
+
+    #[test]
```

Comment:
Does this need a `#[test]` if it just being ignored?

Reply:
Good catch, those are just utilities to set up new fixed test data.